### PR TITLE
Fix PyTorch TransformerModel sample

### PR DESCRIPTION
**Pull request by thiagocrepaldi**
Update PyTorch TranformerModel to be used as sample

### CODE REVIEW

**Review by natke — APPROVED** [review]

File: samples/python/pytorch_transformer/ort_train.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import argparse
```
Comment (spandantiwari) [inline samples/python/pytorch_transformer/ort_train.py:1]: From the diff these seem like totally new files and not updates. :-)
Is that how it is supposed to be?

**Review by BowenBao — APPROVED** [review]

File: samples/python/pytorch_transformer/pt_train.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import argparse
+import math
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
```
Comment (spandantiwari) [inline samples/python/pytorch_transformer/pt_train.py:5]: Checking if nn.Functional needed here?

File: samples/python/pytorch_transformer/pt_train.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    if not args.no_cuda and torch.cuda.is_available():
+        device = "cuda"
+    else:
+        device = "cpu"
+    torch.manual_seed(args.seed)
+
+    # Model
+    criterion = nn.CrossEntropyLoss()
+    lr = 0.001
+    model = TransformerModel(28785, 200, 2, 200, 2, 0.2).to(device)
```
Comment (spandantiwari) [inline samples/python/pytorch_transformer/pt_train.py:74]: minor: It is helpful to capture what these numbers are (which model variable are they feeding into) as a comment for ease of readability in samples.

**Review by spandantiwari — APPROVED** [review]
LGTM.

**Pull request merged**